.github/workflows/ci.yml

```diff
@@ -261,7 +261,7 @@ jobs:
           _make PLATFORM=versal CFG_VERSAL_FPGA_DDR_ADDR=0x40000000
           _make PLATFORM=corstone1000
           _make PLATFORM=nuvoton
-          _make PLATFORM=d06
+          _make PLATFORM=d06 CFG_HISILICON_ACC_V3=y
```

Comment:
Are there `d06` platforms not relying on `CFG_HISILICON_ACC_V3=y`? If so, prefer append a new make command below.